### PR TITLE
Armor vest (Accessory)  - Best grammar and no conflicts (Maybe)

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Changelogs]:
Adds in many different types of vest for each department to help people better survive in there 100% safe work!
Adds in a new node having more advance protection for sec!
List of vest added
Illegal Vest -Admin only ATM
Biohazard resistant vest
Bomb resistant vest
Fire resistant vest
Lava Land Vest
Miner Vest
Ballistic vest -Civ
Close Quarter Combat Vest -Civ
Military grade physal armor vest - Sec
[why]:
Accessory slot is never used actively unless your a miner, lets change that!

### CODE REVIEW

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -254,3 +254,17 @@
...
+	value = -2 // It can hardstun you. You can be a job that your phobia targets...
+	gain_text = "<span class='danger'>You feel your fears manifest themselfs.</span>"
+	lose_text = "<span class='notice'>You feel your fears fade away.</span>"
+	medical_record_text = "Patient has an extreme or irrational fear of or aversion to something."
+	var/datum/brain_trauma/mild/phobia/phobia
+	
+/datum/quirk/phobia/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	phobia = new
+	H.gain_trauma(phobia, TRAUMA_RESILIENCE_SURGERY)
```
Comment (deathride58) [inline code/datums/traits/negative.dm]: ???????????????

File: code/controllers/subsystem/traumas.dm [hunk 2]
```diff
@@ -132,19 +135,26 @@ SUBSYSTEM_DEF(traumas)
...
+					   /obj/item/nullrod/scythe/vibro, /obj/item/energy_katana, /obj/item/toy/katana, /obj/item/nullrod/claymore/katana, /obj/structure/window/paperframe, /obj/structure/mineral_door/paperframe)),
+
+					"mimes" = typecacheof(list(/obj/item/pda/mime, /obj/item/clothing/under/rank/mime, /obj/item/clothing/mask/gas/mime,
+						/obj/item/clothing/head/frenchberet, /obj/item/clothing/suit/suspenders, /obj/item/reagent_containers/food/drinks/bottle/bottleofnothing,
+						/obj/item/storage/backpack/mime, /obj/item/reagent_containers/food/snacks/grown/banana/mime,
+						/obj/item/grown/bananapeel/mimanapeel, /obj/item/cartridge/virus/mime, /obj/item/clothing/shoes/sneakers/mime,
+						/obj/item/bedsheet/mime, /obj/item/reagent_containers/food/snacks/burger/mime, /obj/item/clothing/head/beret, /obj/item/clothing/mask/gas/sexymime,
+						/obj/item/clothing/under/sexymime, /obj/item/toy/figure/mime, /obj/item/toy/crayon/mime, /obj/item/mecha_parts/mecha_equipment/weapon/ballistic/silenced, /obj/mecha/combat/reticence))
```
Comment (deathride58) [inline code/controllers/subsystem/traumas.dm]: ???????????

**Review by deathride58 — CHANGES_REQUESTED** [review]
There's undocumented changes. Did you commit to the wrong branch?

**Comment by Poojawa** [comment]
@deathride58 they PR'd with their master branch

File: code/modules/clothing/under/armor.dm [hunk 3]
```diff
@@ -0,0 +1,91 @@
+/obj/item/clothing/accessory/synd_vest
+	name = "Illegal Vest"
+	desc = "A illegal vest with a mix of different metals and plastics."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: a,e,i,o,u

all of these get "an" instead of 'a'

File: code/modules/clothing/under/armor.dm [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+/obj/item/clothing/accessory/fire_vest
+	name = "Fire resistant vest"
+	desc = "A vest that can be placed over normal clothing to help protect against fire.."
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 0)
+
+/obj/item/clothing/accessory/lava_land_vest
+	name = "Lava Land Vest"
+	desc = "A vest that was made to help protect miners from danger form all sorts of hazards. There are countless labels saying not to wash the vest with any acids."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: "A vest that was designed to protect miners from a variety of hazards. A label advises against acid based cleaning agents."

File: code/modules/clothing/under/armor.dm [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 5,"energy" = 5, "bomb" = 20, "bio" = 30, "rad" = 5, "fire" = 25, "acid" = -45)
+
+/obj/item/clothing/accessory/miner_vest
+	name = "Miner Vest"
+	desc = "A advanced vest that was made to help protect miners from danger from all sorts of hazards. This one is acid-proof."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: "An advanced protective vest designed for mining hazard mitigation."

It's not actually acid proof, just more resistant.

File: code/modules/clothing/under/armor.dm [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 5,"energy" = 5, "bomb" = 35, "bio" = 35, "rad" = 10, "fire" = 35, "acid" = 10)
+
+/obj/item/clothing/accessory/ballistic_vest
+	name = "Ballistic vest"
+	desc = "A flimsy used to help protect against bullets. It's better then nothing... "
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: "A thin lining of Kevlar derivatives, designed to mitigate low power kinetic energy impacts."

File: code/modules/clothing/under/armor.dm [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 5, "bullet" = 10, "laser" = 0,"energy" = 0, "bomb" = 5, "bio" = 5, "rad" = 0, "fire" = 0, "acid" = 5)
+
+/obj/item/clothing/accessory/melee_vest
+	name = "Close Quarter Combat Vest"
+	desc = "A flimsy vest with some plastic to protect against minor physical harm... "
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: "A thin lining of Kevlar derivatives, designed to mitigate low power brute force trauma."

File: code/modules/clothing/under/armor.dm [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 10, "bullet" = 5, "laser" = 0,"energy" = 0, "bomb" = 5, "bio" = 5, "rad" = 0, "fire" = 0, "acid" = 5)
+
+/obj/item/clothing/accessory/sec_melee_vest
+	name = "Military grade physal armor vest"
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: "Surplus Military physical armor vest"

File: code/modules/clothing/under/armor.dm [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+	desc = "A well made vest that helps protect against physical harm. NT still cuts corners when they can and uses less then optimal materials."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: "A surplus military armor lining mean for brute force mitigation."

File: code/modules/clothing/under/armor.dm [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 15, "bullet" = 5, "laser" = 0,"energy" = 0, "bomb" = 15, "bio" = 5, "rad" = 0, "fire" = 0, "acid" = 15)
+
+/obj/item/clothing/accessory/sec_ballistic_vest
+	name = "Military grade bullet armor vest"
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: "Surplus Military flak vest"

File: code/modules/clothing/under/armor.dm [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+	desc = "A well made vest that helps protect against bullets. In the space days its still as basic as it always was."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: "A surplus military lining for low power ballistic protection."

File: code/modules/clothing/under/armor.dm [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 5, "bullet" = 15, "laser" = 0,"energy" = 0, "bomb" = 15, "bio" = 5, "rad" = 0, "fire" = 0, "acid" = 15)
+
+/obj/item/clothing/accessory/energys_vest
+	name = "Laser Dispersion Vest"
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm:73]: Energy dampening vest

File: code/modules/clothing/under/armor.dm [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+	desc = "A vest made to protect and disperse the power of lasers without reflection."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm:74]: A vest designed to mitigate low powered energy weapon impacts

File: code/modules/clothing/under/armor.dm [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 15,"energy" = 10, "bomb" = 5, "bio" = 5, "rad" = 10, "fire" = 0, "acid" = 25)
+
+/obj/item/clothing/accessory/rad_vest
+	name = "Radiation vest"
+	desc = "Heavy plastic padding with some plasma to greatly reduce radiation."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: A flammable vest designed to resist radiation.

File: code/modules/clothing/under/armor.dm [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+/obj/item/clothing/accessory/bomb_vest
+	name = "Bomb resistant vest"
+	desc = "A heavy vest that can be attached to a jumpsuit to help protect against a blast."
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 40, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+
+
+/obj/item/clothing/accessory/fire_vest
+	name = "Fire resistant vest"
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: Flame Retardant vest

File: code/modules/clothing/under/armor.dm [hunk 16]
```diff
@@ -0,0 +1,91 @@
...
+	desc = "A vest that can be placed over normal clothing to help protect against fire.."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: A vest designed to improve the fire resistance of clothing.

File: code/modules/clothing/under/armor.dm [hunk 17]
```diff
@@ -0,0 +1,91 @@
...
+/obj/item/clothing/accessory/bio_vest
+	name = "Biohazard resistant vest"
+	desc = "Biohazard resistant vest that is easily attached to a jumpsuit"
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 40, "rad" = 0, "fire" = 0, "acid" = 10)
+
+/obj/item/clothing/accessory/bomb_vest
+	name = "Bomb resistant vest"
+	desc = "A heavy vest that can be attached to a jumpsuit to help protect against a blast."
```
Comment (Poojawa) [inline code/modules/clothing/under/armor.dm]: A heavy, padded lining designed to mitigate explosive pressure.

**Review by deathride58 — DISMISSED** [review]
The spelling errors add to the charm

Can be merged with kev's approval

**Review by Poojawa — APPROVED** [review]

File: code/modules/research/designs/vest_designs.dm [hunk 18]
```diff
@@ -0,0 +1,120 @@
...
+	desc = "A surplus military armor lining mean for brute force mitigation."
+	id = "sec_ballistic_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 7500, MAT_GLASS = 5500, MAT_GOLD = 2250, MAT_TITANIUM = 5500, MAT_DIAMOND = 4000)
+	build_path = /obj/item/clothing/accessory/sec_ballistic_vest
+	category = list("Misc")
+	departmental_flags =  DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/power_vest
+	name = "Engery protection padding"
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm]: Energy

File: code/modules/research/designs/vest_designs.dm [hunk 19]
```diff
@@ -0,0 +1,120 @@
...
+	desc = "BioHazer resistent vest that is easly attached to a jumpsuit."
+	id = "bio_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1500, MAT_GLASS = 1000, MAT_GOLD = 1750)
+	build_path = /obj/item/clothing/accessory/bio_vest
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/bomb_vest
+	name = "Bomb resitsent vest"
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm]: resistant

**Review by Poojawa — COMMENTED** [review]
just these two missed things, then we're all good. <3

File: code/modules/clothing/under/armor.dm [hunk 20]
```diff
@@ -0,0 +1,91 @@
...
+
+/obj/item/clothing/accessory/fire_vest
+	name = "Flame Retardant vest"
+	desc = "A vest designed to improve the fire resistance of clothing."
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 0)
+
+/obj/item/clothing/accessory/lava_land_vest
+	name = "Lava Land Vest"
```
Comment (ghost) [inline code/modules/clothing/under/armor.dm]: Lavaland

File: code/modules/clothing/under/armor.dm [hunk 21]
```diff
@@ -0,0 +1,91 @@
...
+/obj/item/clothing/accessory/rad_vest
+	name = "Radiation vest"
+	desc = "A flammable vest designed to resist radiation."
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 15, "bomb" = 15, "bio" = 5, "rad" = 50, "fire" = -50, "acid" = 25)
+
+/obj/item/clothing/accessory/power_vest
+	name = "Energy protection padding."
+	desc = "Padding used on suits to try to prevent shocks form being as lethal."
```
Comment (ghost) [inline code/modules/clothing/under/armor.dm]: Form -> From

File: code/modules/research/designs/vest_designs.dm [hunk 22]
```diff
@@ -0,0 +1,120 @@
+/datum/design/bio_vest
+	name = "BioHazer resistent vest"
+	desc = "BioHazer resistent vest that is easly attached to a jumpsuit."
```
Comment (ghost) [inline code/modules/research/designs/vest_designs.dm]: Biohazard

File: code/modules/research/designs/vest_designs.dm [hunk 23]
```diff
@@ -0,0 +1,120 @@
...
+	id = "bio_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1500, MAT_GLASS = 1000, MAT_GOLD = 1750)
+	build_path = /obj/item/clothing/accessory/bio_vest
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/bomb_vest
+	name = "Bomb resistsent vest"
```
Comment (ghost) [inline code/modules/research/designs/vest_designs.dm]: Resistant

File: code/modules/research/techweb/all_nodes.dm [hunk 24]
```diff
@@ -602,6 +602,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/sec_vests
+	id = "sec_vests"
+	display_name = "Military Vests"
+	description = "High grade protection vests for Civ and Military presents.."
```
Comment (ghost) [inline code/modules/research/techweb/all_nodes.dm]: "for civilian and military presence"?

**Review by ghost — CHANGES_REQUESTED** [review]
You're getting better at the spelling but I caught a few

File: code/modules/research/designs/vest_designs.dm [hunk 25]
```diff
@@ -1,6 +1,6 @@
 /datum/design/bio_vest
-	name = "BioHazer resistent vest"
-	desc = "BioHazer resistent vest that is easly attached to a jumpsuit."
+	name = "BioHazerd resistent vest"
```
Comment (deathride58) [inline code/modules/research/designs/vest_designs.dm:2]: Biohazard

File: code/modules/research/designs/vest_designs.dm [hunk 26]
```diff
@@ -9,7 +9,7 @@
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_MEDICAL
 
 /datum/design/bomb_vest
-	name = "Bomb resistsent vest"
+	name = "Bomb resistantt vest"
```
Comment (deathride58) [inline code/modules/research/designs/vest_designs.dm:12]: resistant

**Comment by Trilbyspaceclone** [comment]
There in Datum and I somehow don't have the branch anymore.

**Pull request closed without merging**